app/views/shortlists/create_shortlist.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Shortlists#create_shortlist</h1>
```

Comment:
i dont actually think that we need a view for the shortslists#create action because its only happening while the user is on a different site like show or index.